scripts/github_pr/github_pr_cloud.yaml

```diff
@@ -46,6 +47,26 @@ template:
             parameters:
               status: error
               message: Owner of the source repo for this PR lacks permission
+      - type: FileMatch
+        config:
+          pass_through: all
+          paths:
+            -  !ruby/regexp '/scripts\/jenkins\/ardana\//'
+        whitelist_handler:
+          - type: JenkinsJobTriggerArdana
+            parameters:
+              detail_logging: true
+              job_name: openstack-ardana
+              job_cmd: "../jenkins/jenkins-job-trigger"
+              job_parameters:
+                <<: *ardana_automation_parameters
+        blacklist_handler:
+          - type: SetStatus
+            config:
```

Comment:
@jdsn my ruby is not very strong, will setting the config.context object here automagically work, or will setting a unique context value per SetStatusAction require changes to github-pr?